x-pack/solutions/observability/plugins/slo/server/services/find_slo_definitions.ts

```diff
@@ -22,8 +22,17 @@ export class FindSLODefinitions {
   constructor(private repository: SLORepository) {}
 
   public async execute(params: FindSLODefinitionsParams): Promise<FindSLODefinitionsResponse> {
+    let requestTags: string[] = [];
+
+    if (Array.isArray(params.tags)) {
+      requestTags = params.tags;
+    } else if (params.tags) {
+      requestTags = [params.tags];
+    }
+
```

Comment:
If you keep it a coma separated list, you can simplify this to just a `const tags = params.tags.split(",")`